apps/server/src/modules/provisioning/strategy/sanis/schulconnex-response-mapper.ts

```diff
@@ -172,6 +182,57 @@ export class SchulconnexResponseMapper {
 		return mapped;
 	}
 
+	private static mapGroupDuration(duration: SchulconnexLaufzeitResponse | undefined): TimePeriode | undefined {
+		if (!duration) {
+			return undefined;
+		}
+
+		if (duration.von && duration.bis && duration.bis > duration.von) {
```

Comment:
duration.bis > duration.von more looks like a validation for me. Maybe we can write our own validationConstraint which we puts to the schulconnex-laufzeit-response.ts ? https://github.com/typestack/class-validator?tab=readme-ov-file#custom-validation-classes

Reply:
Validations throw errors and that is something I wanted to avoid. If the validation should not be in the mapper then we should remove it completly.